batch/2017-01-01.4.0/examples/NodeAddUser.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "api-version": "2017-01-01.4.0",
+    "poolId": "poolId",
+    "nodeId": "tvm-1695681911_1-20161121t182739z",
+    "client-request-id": "00000000-0000-0000-0000-000000000000",
+    "ocp-data": "Fri, 17 Feb 2017 00:00:00 GMT",
+    "User": {
+      "name": "userName",
+      "isAdmin": false,
+      "expiryTime": "0001-01-01T00:00:00Z",
```

Comment:
Is this the magic value for 'no/default expiry'?  If not, maybe use a more realistic time?